packages/instantsearch-react/src/translatable.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+export default function translatable(defaultTranslations) {
+  return Composed => {
+    function Translatable(props) {
+      const {translations, ...otherProps} = props;
+      const translate = (key, ...params) => {
+        const translation =
+          translations && {}.hasOwnProperty.call(translations, key) ?
```

Comment:
I don't mind either way, but if we do choose to use object prototypes builtins directly in our projects, we should update the [algolia eslint config](https://github.com/algolia/eslint-config-algolia/blob/master/index.js#L50).